Library/Formula/ga.rb

```diff
@@ -0,0 +1,60 @@
+class Ga < Formula
+  homepage "http://hpc.pnl.gov/globalarrays/"
+  url "http://hpc.pnl.gov/globalarrays/download/ga-5-3.tgz"
+  version "5.3"
+  sha1 "b4e5fdc7f25e76d4d9e4c8b23f9c9ae2855cd925"
+
+  depends_on "gcc"
```

Comment:
Use `fails_with` here instead. Note that we'll probably not include this if it doesn't build with Clang.